### ISSUE TITLE
Advanced Search Selectors - 3.3.2 Labels or Instructions - 7 missing and 6 empty form labels within the Advanced Search tab

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#10052]
BUG RAISED AS A PLACEHOLDER - RESEARCH ON WHETHER A FIX IS NECESSARY TBC.

As a user reliant on the screen reader to collect information about the form fields in a page, I expect every field in the page to have a label associated with it which can be interpreted by the screen reader. Labels should contain information about the name and purpose of the field. 

**Linked Work Item:**
#9492 

**Expected Result:**
All fields should have labels in the html which contain information about the name and function of the form fields. This is to allow the screen reader to read out information about the name and purpose of the field to users.

**Actual Result:**
7 missing and 6 empty form labels within the Advanced Search tab, appearing on various - but not all - select2 dropdowns. Fields with missing form labels do not have any `label` `/label` operators in the html which can contain information about the dropdown for screen reader users. Fields with empty form labels have the operator present in the html which can contain information about the field, but no information is contained within it. Despite this, the screen reader is still able to convey the name and purpose of these fields to the user [whether the labels should also be present in the html is to be confirmed by @imogenroberts]

**Evidence:**

The following are examples of the various missing an empty form labels in the Advanced Search Selectors Pane.

![Image](https://github.com/archesproject/arches/assets/77161085/ed5876fe-a021-41c4-9089-4e90a48e27e0)
(In this image, a `label` operator is present, but contains no meaningful information about the name or purpose of the form field)

![Image](https://github.com/archesproject/arches/assets/77161085/d3391a12-fd83-4082-b1e3-ef256f3e006a)
(No `label` operator is present in the html for this select2 dropdown - this issue is repeated on all other fields missing a form label)

**Comment by johnatawnclementawn** [comment]
@imogenroberts this should be resolved in the latest

**Comment by imogenroberts** [comment]
RETEST: All missing and empty form labels are now resolved.

**Evidence:**
![Advanced Search Panel - 3 3 2 7 missing 6 empty form labels FIXED](https://github.com/archesproject/arches/assets/77161085/c1b30c67-e6e1-4123-adae-cc76eea90ab2)
